README.md

```diff
@@ -47,53 +50,62 @@ You can now create, drop any database models. The created model is **User**, und
 - Also, feel free to modify or remove some items to meet your expectation/demand
 
 # Folder Structure (More updates expected)
-    ├── src                   
-    │   ├─ config  
+
+    ├── src
+    │   ├─ config
+    |       ├─ index.js
+    │   ├─ controllers
     |       ├─ index.js
-    │   ├─ controllers  
+    |       ├─ user.js
+    │   ├─ middlewares
     |       ├─ index.js
     |       ├─ user.js
-    │   ├─ middlewares      
-    |       ├─ index.js  
-    |       ├─ user.js  
-    │   ├─ models          
+    │   ├─ models
     |       ├─ index.js
-    |       ├─ user.js 
-    |       ├─ setup.js 
-    |   └─ routers 
+    |       ├─ user.js
+    |       ├─ setup.js
+    |   └─ routers
     |       ├─ index.js
-    |       ├─ user.js  
-    |   └─ scripts 
+    |       ├─ user.js
+    |   └─ scripts
     |       ├─ createdb.js
-    |       ├─ dropdb.js  
-    |       ├─ index.js  
-    ├── test                  
-    │   ├─ index.js           
-    │   
+    |       ├─ dropdb.js
+    |       ├─ index.js
+    ├── test
+    │     ├─ controllers
+    │         ├─ index.js
+    │         ├─ user.js
+    │     ├─ middleware
+    │         ├─ index.js
+    │     ├─ routes
+    │         ├─ index.js
+    │     └─ index.js
+    │
     └── index.js
```

Comment:
I will fix this in the next PR. I think `index.js` should be inside `src/`

Reply:
Alright, thanks.